modules/scheduler/impl/servicegroup/delete.go

```diff
@@ -30,21 +32,30 @@ func (s ServiceGroupImpl) Delete(namespace string, name, force string) error {
 	if err := s.js.Get(context.Background(), mkServiceGroupKey(namespace, name), &sg); err != nil {
 		if force != "true" {
 			if err == jsonstore.NotFoundErr {
+				logrus.Errorf("not found runtime %s on namespace %s", name, namespace)
 				return DeleteNotFound
 			}
+			logrus.Errorf("get from etcd err: %v when delete runtime %s on namespace %s", err, name, namespace)
 			return err
 		}
 	}
-
+	ns := sg.ProjectNamespace
+	if ns == "" {
+		ns = sg.Type
+	}
+	logrus.Infof("start to delete service group %s on namespace %s", sg.ID, ns)
 	if _, err := s.handleServiceGroup(context.Background(), &sg, task.TaskDestroy); err != nil {
 		if force != "true" {
 			return err
 		}
 	}
+	logrus.Infof("delete service group %s on namespace %s successfully", sg.ID, ns)
+	logrus.Infof("delete service group %s on namespace %s from etcd", sg.ID, ns)
```

Comment:
no need to write two lines of log

Reply:
Done. remove `logrus.Infof("delete service group %s on namespace %s successfully", sg.ID, ns)`